main/main.cpp

```diff
@@ -4117,6 +4132,11 @@ int Main::start() {
 
 	if (!script.is_empty()) {
 		Ref<Script> script_res = ResourceLoader::load(script);
+#ifndef TOOLS_ENABLED
+		if (standalone_script && script_res.is_null()) {
+			OS::get_singleton()->alert(vformat("Can't load the script \"%s\".", script));
+		}
+#endif
 		ERR_FAIL_COND_V_MSG(script_res.is_null(), EXIT_FAILURE, "Can't load script: " + script);
```

Comment:
Make the message identical to the alert:

```suggestion
		ERR_FAIL_COND_V_MSG(script_res.is_null(), EXIT_FAILURE, vformat("Can't load the script \"%s\".", script));
```

Also, since `alert()` always prints a message to the command line, you can probably remove the error printing and just use `if (script_res.is_null()) { return EXIT_FAILURE; }`.